website/releases/download.md

```diff
@@ -6,35 +6,62 @@ toc: true
 last_modified_at: 2019-12-30T15:59:57-04:00
 ---
 
-## Release 0.9.0
+### Release 0.9.0
 * Source Release : [Apache Hudi 0.9.0 Source Release](https://www.apache.org/dyn/closer.lua/hudi/0.9.0/hudi-0.9.0.src.tgz) ([asc](https://downloads.apache.org/hudi/0.9.0/hudi-0.9.0.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.9.0/hudi-0.9.0.src.tgz.sha512))
```

Comment:
I assume mirror is a requirement only for .src.tgz and not asc and sha files ? Just wanted to confirm.

Reply:
I had checked for other projects which were accepted recently: https://tika.apache.org/download.html and https://apisix.apache.org/downloads/ . Only `.src.tgz` refers to the mirror.